pandas/tests/generic/test_generic.py

```diff
@@ -431,7 +431,7 @@ def test_size_compat(self):
 
     def test_split_compat(self):
         # xref GH8846
-        o = self._construct(shape=10)
+        o = self._construct(shape=10).values
         assert len(np.array_split(o, 5)) == 5
```

Comment:
nparray_split depends on np.swapaxes, so I sidestep the problem here. This test is not very useful. Is that ``__array_function__`` in the pipeline to implement?

I'm thinking that maybe we should maybe keep swapaxes, perhaps hidden in ``_deprecations`` like we do with ``tolist``?